wv2util/MainWindow.xaml.cs

```diff
@@ -336,5 +355,90 @@ private void WatchForChangesTimer_Elapsed(object sender, ElapsedEventArgs e)
             }
             m_previousHostAppEntries = currentHostAppEntries;            
         }
+
+        private string newsLink = "https://api.github.com/repos/MicrosoftEdge/WebView2Announcements/issues";

```

Comment:
This can be static and static members start s_
```suggestion
        private static string s_newsLink = "https://api.github.com/repos/MicrosoftEdge/WebView2Announcements/issues";
```